pipelines/utilities/longhorn_status.sh

```diff
@@ -1,10 +1,13 @@
 wait_longhorn_status_running(){
-  local RETRY_COUNTS=60 # in minutes
+  local RETRY_COUNTS=10 # in minutes
```

Comment:
Will this `RETRY_COUNTS` affect the test case? I assume the timeout set in https://github.com/longhorn/longhorn-tests/pull/2573/files#diff-360108ce1e2de83204c6e79a21f402be4d6a9abd6f4b128e855c749bf2e68f19R18 is sufficient for running the test case?

Reply:
Reverted the change. The idea is simply to align the timeout between the shell script and the Python function.

Reply:
Hi @yangchiu, I have updated and ready for review, thank you.